src/styles/styles.scss

```
@@ -0,0 +1,29 @@
+@import "./Roboto/fonts";
+
+html,
+body {
+  font-family: "Roboto";
```

```suggestion
  font-family: "Roboto", sans-serif;
```

чтобы браузер отображал другой шрифт без засечек, пока Roboto не загрузится (или если вообще не загрузится по какой-то причине)